OMC56/Thor Method_w_JP_comments.py

```diff
@@ -0,0 +1,69 @@
+# ************************
```

Comment:
Combine comments from JP so there's just one `Thor method.py` file will all improvements made to it. Delete this file.